R/input-switch.R

```diff
@@ -45,6 +45,7 @@
 input_switch <- function(id, label, value = FALSE, width = NULL) {
   tag <- input_checkbox(id, label, class = "form-check form-switch", value = value, width = width)
   tag <- tag_require(tag, version = 5, caller = "input_switch()")
+  tag <- attachDependencies(tag, component_dependency_js("toggleInputBinary"))
```

Comment:
This is a minor sidenote, but I tend to prefer including a `htmlDependency()` as a child rather than an attribute.

And, to that point, maybe it makes sense to include the dependency as a child of `input_checkbox()`?

That said, I'm ok with keeping it the way it is (up to you)

Reply:
Agreed on both accounts! I moved the dependency into `input_checkbox()`.